docs/content/preview/api/ysql/the-sql-language/statements/cmd_copy.md

```diff
@@ -143,3 +143,11 @@ The following copy options may help to speed up copying, or allow for faster rec
 * `DISABLE_FK_CHECK` skips the foreign key check when copying new rows to the table.
 * `REPLACE` replaces the existing row in the table if the new row's primary/unique key conflicts with that of the existing row.
 * `SKIP n` skips the first `n` rows of the file. `n` must be a nonnegative integer.
+* `ROWS_PER_TRANSACTION n` split the copy process into many smaller transactions (rather than one large transaction) (default 20000). 
```

Comment:
Can we please add the defaults for other variables as well and also some examples which uses these options.

DISABLE_FK_CHECK -- default false, i.e FK_check is done
REPLACE -- by default conflict error is reported.
SKIP -- Default N == 0

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/13677/commits/ee838f7f196e18561fd52f2806e9e9f5e26be6d3